docker-compose.production.yml

```diff
@@ -17,6 +18,13 @@ services:
       - SESSION_SECRET
       - WEBAPP_ORIGIN=https://rhi.zone
     image: ghcr.io/opentree-education/rhizone-lms_api:latest
+  flyway:
+    command: -configFiles=/run/secrets/FLYWAY_CONFIG migrate
+    deploy:
+      replicas: 1
+    image: ghcr.io/opentree-education/rhizone-lms_flyway:latest
+    secrets:
+      - FLYWAY_CONFIG
```

Comment:
The flyway config will be managed as a file:

https://flywaydb.org/documentation/configuration/configfile.html

Using Docker Swarm Secrets:

https://docs.docker.com/engine/swarm/secrets/